docker/build.Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && export TZ=America/New_York \
     && wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.216-bionic.list https://packages.lunarg.com/vulkan/1.3.216/lunarg-vulkan-1.3.216-bionic.list \
     && apt-get update \
     && apt-get install -yqq --no-install-recommends \
-    vulkan-sdk x11-xserver-utils libvulkan-dev libvulkan1 xvfb mesa-vulkan-drivers \
+    vulkan-sdk x11-xserver-utils libvulkan-dev libvulkan1 xvfb mesa-vulkan-drivers llvm-11-dev \
```

Comment:
nit: Could you move this together with other llvm tool a few lines above?

Reply:
I did it first but I could install llvm-dev only after the apt-get update. That's why I put it here.

Reply:
I've just checked and it works

Reply:
I added it just after lld-11